qutebrowser/config/configdata.yml

```diff
@@ -1541,8 +1541,14 @@ zoom.text_only:
 
 colors.completion.fg:
   default: white
-  type: QtColor
-  desc: Text color of the completion widget.
+  type:
+    name: ListOrValue
+    valtype: QtColor
+  desc: >-
+    Text color of the completion widget.
+
+    May be a single value to specify the color for all columns, or a list
+    specifying a different color for each column.
```

Comment:
Maybe the number of columns (three) should be mentioned here.
What happens if I specify four (I guess the last one is simply ignored) or just two (I guess the first one is recycled) colors in the list?